config/gflownet/gflownet.yaml

```diff
@@ -19,8 +19,11 @@ optimizer:
   adam_beta2: 0.999
   # Momentum for SGD
   sgd_momentum: 0.9
-  # Mini-batch size
-  batch_size: 10
+  # Number of trajectories of each kind
+  batch_size:
+    forward: 10
```

Comment:
Could you clarify what are the specific batch sizes (forward, train, replay)?

Reply:
I explain that a bit in the PR description but I should add some readme info (where though?). In short:

- forward: number of forward trajectories to include in the training batch. These are on-policy trajectories possibly with random actions (if `random_action_prob > 0`) or with a tempered policy if `temperature < 1.0`
- train: number of backward trajectories to include in the training batch, sampled (backwards) from data points in a "training set"
- replay: number of backward trajectories to include in the training batch, sampled (backwards) from data points in the replay buffer.

The total number of trajectories in the training batch is the sum of the above.

Reply:
I see, how about renaming `train` to something like `backward_dataset`, and `replay` to something like `backward_replay`?

And yeah, I meant some readme/docs - I know that at this point there are more urgent things to do, so how about just creating an issue `Create training README`, and slowly start dumping ideas for what needs to be included there (I think they come up periodically during code review), and eventually at some point actually using this to create some doc?

Reply:
Also, maybe just add some small comments (essentially paste what you wrote here) in the `config/gflownet/gflownet.yaml` itself?

Reply:
Sounds good! Actually I wanted to start writing docs as I coded but I have never done something like that in a big project, so I am not sure what the best way is. If you know about all this, I'd be very happy to discuss.

Reply:
Done [here](https://github.com/alexhernandezgarcia/gflownet/pull/150/commits/897ffe32ca84a131f5ffe5b69c72f9dcefc32f4e).